recengine/recengine.py

```diff
@@ -53,14 +53,15 @@ def __init__(self, row):
         self.exercise = int(row[0])
         self.percent_1rm = float(row[1])
         self.repetitions = int(row[2])
-        self.date = datetime.strptime(row[3], "%m/%d/%Y %H:%M").date()
+        self.datetime = datetime.strptime(row[3], "%m/%d/%Y %H:%M")
```

Comment:
Do we need both date and datetime? Just using datetime should be possible

Reply:
You are 100% correct. Fixed it.